cookbooks/fastscape_eroding_box.prm

```diff
@@ -0,0 +1,172 @@
+# At the top, we define the number of space dimensions we would like to
+# work in:
+set Dimension                              = 3
+
+set Use years in output instead of seconds = true
+set End time                               = 2.5e5
+set Maximum time step                      = 50000 #5e6
+set Output directory                       = eroding_box
+
+set Pressure normalization                 = no #surface
+set Surface pressure                       = 0
+
+# A box which has one half raised by 1 km.
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X extent = 100e3
+    set Y extent = 100e3
+    set Z extent = 25e3
+    set X repetitions = 4
+    set Y repetitions = 4
+  end
+  
+  subsection Initial topography model
+    set Model name = function
+    subsection Function
+      set Function expression = if(y>20e3 && y<80e3 && x>20e3 && x<80e3, 2500, 0)          
+    end
+  end
+end
+
+# We do not consider temperature in this setup
+subsection Initial temperature model
+  set Model name = function
+
+  subsection Function
+    set Function expression = 0
+  end
+end
+
+subsection Boundary temperature model
+  set Fixed temperature boundary indicators = bottom, top
+  set List of model names = initial temperature
+end
+
+# X boundaries are periodic, and the rest are tangential.
+subsection Boundary velocity model
+  set Tangential velocity boundary indicators = bottom, front, back, left, right
+end
+
+# Here we setup the top boundary with fastscape.
+subsection Mesh deformation
+  set Mesh deformation boundary indicators = top : fastscape
+  
+    subsection Fastscape
+      # As the highest resolution at the surface is 4 in the mesh refiniment function, we set this the same.
+      set Surface resolution = 4
+
+      # Because we only have 1 level of refinement at the surface we set this to 0.
+      # If we had multiple resolutions, e.g. 3 different cell sizes at the surface,
+      # this would be set to 2 (number of cell sizes at surface - 1).
+      set Resolution difference = 0
+
+      # We set fastscape as 1 levels more resolved than ASPECT's initial global surface resolution.
+      set Additional fastscape refinement = 1
+
+      # Number of FastScape timesteps we want per ASPECT timestep.
+      set Number of steps = 5
+
+      # Set the maximum allowable FastScape timestep. If the (ASPECT timestep)/(Number of steps)
+      # is greater than this value, then the number of steps is automatically doubled.
+      # This is continued until the FastScape timestep is less than this value.
+      set Maximum timestep = 10000
+
+      # Seed number for initial topography noise
+      set Fastscape seed = 1000
+
+      # Because no boundaries are periodic, no flux, and no advection we don't need ghost nodes.
```

Comment:
```suggestion
      # Because no boundaries are periodic, no flux is prescribed, and no advection occurs in FastScape, we do not need ghost nodes.
```